### PR TITLE
Adds new sounds when sweeping and mopping

### PR DESCRIPTION
**Pull request by Henri215**
## What Does This PR Do

Adds new sounds when sweeping and mopping

Sweeping sound taken from https://freesound.org/people/georgesavic/sounds/593466/
Mopping sound taken from https://freesound.org/people/blwuens/sounds/515158/
## Why It's Good For The Game

A better and non generic sound for the push broom is good to have.
The mopping sound, that has a cooldown, could add to the atmosphere of being a janitor.
## Images of changes

https://user-images.githubusercontent.com/77684085/222933443-32c684a4-e76f-4cc1-8d5a-5524801ee3dd.mp4

## Testing

- See video above
## Changelog
:cl: Henri237, Carthusia
soundadd: Added new sounds when sweeping and mopping
/:cl:

### CODE REVIEW

File: code/game/objects/items/weapons/mop.dm [hunk 1]
```diff
@@ -14,6 +16,8 @@
 	var/mopcount = 0
```
Comment (GDNgit) [inline code/game/objects/items/weapons/mop.dm:16]: this var isn't used anywhere wtf

**Review by GDNgit — APPROVED** [review]
Code LGTM, perhaps you may want to add a comment for the MOP_SOUND_CD though?

**Comment by Henri215** [comment]
> Code LGTM, perhaps you may want to add a comment for the MOP_SOUND_CD though?

Added a comment

**Comment by Denthamos** [comment]
The mop sounds.....dry?  Only comment I would have on it, otherwise the sweeping is nice.

**Comment by Carthusia** [comment]
> The mop sounds.....dry? Only comment I would have on it, otherwise the sweeping is nice.

Overall with mopping, you dont really hear the floor cleaning sound while you do it, my experience with actual janitorial work has been you hear a sound when you dip it in the bucket, drain the mop of excess and that initial splat once it hits the floor, but besides that its a very light swishing sound.

The sound above is that of a swiffer mop, other mop sounds generally have to much detail in them, lots of splats, splorts and swashes, which for a sound thats going to be repeated alot to alot of people in a large area would cause it to become very grating and annoying over time, kind of like the surgery sounds PR, they just had to much detail and goop to them which would have drove people insane over time repeatedly hearing them.

This sound is fairly low detail and gets the point accross that someone is cleaning, but its not so loud and detailed that it overshadows all other sounds around it.

Its just a simple little mopping que sound to be relegated to the background noise of the station, all the big noises remain front and center since they are far more important for someone to hear.

**Review by hal9000PR — APPROVED** [review]

**Pull request merged**